src/Appwrite/Utopia/Request/Filters/V13.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Appwrite\Utopia\Request\Filters;
+
+use Appwrite\Utopia\Request\Filter;
+
+class V13 extends Filter
+{
+    // Convert 0.12 params format to 0.13 format
+    public function parse(array $content, string $model): array
+    {
+        switch ($model) {
```

Comment:
I think we need one for function entrypoint, right? It was 'code', now it's 'entrypoint'. Wasnt it renamed?

Even if name wasn't changed, we should probably split by space and use the last thing.